lib/sycamore/sycamore/utils/aryn_config.py

```diff
@@ -5,7 +5,9 @@
 
 from typing import Dict, Any
 
-_DEFAULT_PATH = os.path.join(pathlib.Path.home(), ".aryn", "config.yaml")
+DEFAULT_PATH = os.path.join(pathlib.Path.home(), ".aryn", "config.yaml")
+_DEFAULT_PATH = DEFAULT_PATH
+## ToDO: remove after 31/08/2024
```

Comment:
nit:
I'll recommend writing dates as YYYY-MM-DD. Three advantages:
1. its the ISO 8601 standard
2. they sort properly in that order
3. It's sufficiently unambiguous even with days <=12.

Reply:
Makes sense, will use this format in future.